src/osd/PGLog.cc

```diff
@@ -86,24 +86,22 @@ void PGLog::IndexedLog::trim(
       }
     }
 
-    bool reset_complete_to = false;
```

Comment:
We have this check https://github.com/ceph/ceph/blob/master/src/osd/PGLog.cc#L62, to ensure that the log in not empty.